modules/postgres/outputs.tf

```diff
@@ -0,0 +1,48 @@
+output "address" {
+  value = aws_db_instance.main.address
+}
+
+output "arn" {
+  value = aws_db_instance.main.arn
+}
```

Comment:
I would personally remove all the outputs that are not useful for now. So keep: address, username and password